src/main/resources/index.jelly

```diff
@@ -4,5 +4,5 @@
   Since we don't really have anything dynamic here, let's just use static HTML. 
```

Comment:
I withdraw only this change because this file must use only static html.

`
Caused by: org.apache.commons.jelly.JellyException: Unable to create expression: %general.description
	at org.apache.commons.jelly.expression.jexl.JexlExpressionFactory.createExpression(JexlExpressionFactory.java:64)
	at org.apache.commons.jelly.expression.CompositeExpression.parse(CompositeExpression.java:82)
	at org.apache.commons.jelly.parser.XMLParser.addTextScript(XMLParser.java:1165)
	at org.apache.commons.jelly.parser.XMLParser.endElement(XMLParser.java:709)
	... 135 more
`